theme/less/components/tags.less

```diff
@@ -15,8 +15,30 @@ They also come in a bigger size with the `.big` class.
 </ul>
 <span class="big tag">Big tag</span>
 ```
+
+## DSFR tags
+
+We can use DSFR tags and their customizations. See Tabs-js for how to use it as tabs.
+There is also `.fr-tags-group` to group multiple tags together like a row.
+
+```dsfr-tags.html
+<nav class="tabs fr-tags-group">
```

Comment:
I think we can remove `tabs` here?

Reply:
Yes ! Tabs and tab have nothing to do with tags style